test/lib/compiler.test.js

```diff
@@ -29,6 +29,28 @@ describe('Compiler', function () {
     assume(compiler).to.be.instanceof(Compiler);
   });
 
+  describe.only('#catalog', function () {
+    it('will catalog a page (assemble all of the assets) for bootstrap', function (done) {
```

Comment:
`catalog a pagelet`, this is also some legacy. I know the catalog refers to pages, but those do not exists anymore, it are all pagelets. Removal of page was intended to be part of rewriting the compiler.